### PR TITLE
Support autotest stop command

### PR DESCRIPTION
**Pull request by heliclei**

### CODE REVIEW

**Comment by heliclei** [comment]
[ci rebuild]

**Comment by heliclei** [comment]
[ci rebuild]

**Comment by heliclei** [comment]
[ci rebuild]

**Comment by heliclei** [comment]
[ci rebuild]

File: tests/cpp-tests/Classes/controller.cpp [hunk 1]
```diff
@@ -254,6 +255,77 @@ void TestController::onMouseScroll(Event *event)
...
+        send(fd, "\n",1,0);
+        wait(2);
+
+        while(1)
+        {
+            if(_exitThread)
+            {
+                return;
+            }
+            //currentTest->nextCallback(nullptr);
```
Comment (dumganhar) [inline tests/cpp-tests/Classes/controller.cpp:298]: Please remove unused comments.

**Pull request merged**